src/noit_clustering.c

```diff
@@ -449,11 +449,11 @@ repl_work(eventer_t e, int mask, void *closure, struct timeval *now) {
     uuid_unparse_lower(my_id, my_id_str);
     CURL *curl = get_curl_handle();
     if(curl == NULL) {
+      mtevL(clerr, "Can't get curl handle: waiting %ds\n", REPL_FAIL_WAIT_US/1000000);
```

Comment:
I'd suggest making this %f and `(double)REPL_FAIL_WAIT_US/1000000.0`

Reply:
changed